flask_simplelogin/__init__.py

```diff
@@ -180,9 +197,19 @@ def init_app(self, app, login_checker=None,
         if login_form:
             self._login_form = login_form
 
+        # If the user is passing a new dictionary
         if messages and isinstance(messages, dict):
             self.messages.update(messages)
-
+        # If the user is disabling messages
+        # Must differentiate between None and False.
+        elif messages is False:
+            
+            # I almost put this in a dict comprehension, but I couldn't figure out how to update value.enabled
```

Comment:
Please, don't go for dict comprehension. It is not supported in Python 2. I'm in favor of dropping support for Python 2 anytime, but this is a matter for another branch/PR.